backend/src/main/scala/bloop/io/ParallelOps.scala

```diff
@@ -173,9 +174,9 @@ object ParallelOps {
           ()
         } catch {
           case NonFatal(t) =>
-            logger.report(
-              s"Unexpected error when copying $originFile to $targetFile, you might need to restart the build server.",
-              t
+            logger.error(
```

Comment:
What's the difference between `logger.report` and `logger.error`?

Would it be better to add a `def error(msg: String, throwable: Throwable)` method in `Logger` instead?

Reply:
Good idea! I removed reporrt and just changed it to error, this also prompted a bit of refactoring.